modules/nixos/panel.nix

```
@@ -56,6 +79,8 @@ in {
           default = false;
           description = ''
             Create the MySQL database and database user locally, and run installation.
+            The mysql database will be handled declaratively, however do note that the panel
+            settings will only be set at startup, any addtions need to be done in <dataDir>/.env.
```

```suggestion
            Create the MySQL database and database user locally, and run installation.
            
            ::: {.warning}
            The mysql database will be handled declaratively, however do note that the panel
            settings will only be set at startup, any addtions need to be done in
            {option}`services.pterodactyl.dataDir`{file}`/.env`.
            :::
```

Let's use the warning markup here, and for the redis option.